compute_tools/src/http/openapi_spec.yaml

```diff
@@ -83,6 +83,87 @@ paths:
               schema:
                 $ref: "#/components/schemas/DbsAndRoles"
 
+  /promote:
+    post:
+      tags:
+        - Promotion
+      summary: Promote secondary replica to primary
+      description: ""
+      operationId: promoteReplica
+      requestBody:
+        description: Promote requests data
+        required: true
+        content:
+          application/json:
+            schema:
+                $ref: "#/components/schemas/SafekeepersLsn"
```

Comment:
Normally anything we do in terms of updating compute state we do it by sending the spec. What will happen if we only send these things in the body?

Previous spec that was configured will be incorrect. That's a bit confusing.